readme.txt

```diff
@@ -109,6 +109,14 @@ Web Stories are powered by [AMP](https://amp.dev/), which adds some restrictions
 
 For the plugin's full changelog, please see [the Releases page on GitHub](https://github.com/google/web-stories-wp/releases).
 
+= 1.7.2 =
+
+**Release Date:** May 20, 2021.
+
+* New feature: optimize videos directly from the pre-publish checklist.
+* Fixes an issue with the preview mode not working anymore.
```

Comment:
Should we mention the video poster check? "Added pre-publish check for missing video posters."

Reply:
Yes.

At the time I opened this, that feature wasn't on the list yet.